lib/actions-oclif/os/configure.ts

```diff
@@ -213,14 +225,36 @@ export default class OsConfigureCmd extends Command {
 
 		console.info('Configuring operating system image');
 
+		const image = params.image;
 		await helpers.osProgressHandler(
 			await devInit.configure(
-				params.image,
+				image,
 				deviceTypeManifest,
 				configJson || {},
 				answers,
 			),
 		);
+
+		if (options["system-connection"]) {
+			const files = await Bluebird.map(options["system-connection"], async filePath => {
+				const content = await fs.promises.readFile(filePath, 'utf8');
```

Comment:
What the difference between `fs.promises.readFile` and `fs.readFile`? (given `const fs = await import('mz/fs');`)

Reply:
@pdcastro I missed the `mz/fs` import, I'll use that instead to keep the usage consistent